text/0265-event-vision.md

```diff
@@ -0,0 +1,79 @@
+# Event Basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to document and agree on a few basics.
+
+### What are OpenTelemetry Events?
+
+OpenTelemetry Events are a type of OpenTelemetry Log that requires an event name and follows a specific structure implied by that event name.
+
+They are a core concept in OpenTelemetry Semantic Conventions.
+
+### OTLP
+
+Since OpenTelemetry Events are a type of OpenTelemetry Log, they share the same OTLP log data structure and pipeline.
+
+### API
+
+OpenTelemetry should provide a (user-facing) Logs API that includes the capability to emit OpenTelemetry Events.
```

Comment:
Is "should" normative here? For languages implementations that it makes sense to wrap already existing Log APIs can they ignore this recommendation?

Reply:
It looks like it should be "must" as otherwise it would be against what is written a little lower in "Alternatives" section

> These alternatives primarily boil down to differences in [...]
organization (e.g. whether Event API should be something separate from Logs API).

Reply:
IIRC we don't use normative language _as much_ in the OTEPs repo (just FYI).

Reply:
This needs to be made clear what the expectation is.

We can't just assume readers are going to understand this as requirement when we are communicating it as a pseudo-recommendation.

I do not think it should be a requirement. And if that is what is trying to be merged here it needs to be made clear.

Reply:
Given the amount of discussions around Events API vs Logs API, I agree that this section should be clarified. I feel that many reviewers (including myself) accepted the OTEP, because of the conversation and "implicit" agreements made during SIG meetings. These conclusions could be better described.

Reply:
as discussed in Spec SIG meeting, I believe this is resolved now by changing should to SHOULD in https://github.com/open-telemetry/oteps/pull/265/commits/e46600340cdd0344dde4387f63219b2c7459f5a7

resolving this discussion, but please open a new one with summary if there are any additional concerns beyond that, thanks!